Android/ChorusRFLaptimer/app/src/main/java/app/andrey_voroshkov/chorus_laptimer/AppState.java

```diff
@@ -607,6 +608,11 @@ public void addLapResult(int deviceId, int lapNumber, int lapTime) {
             return;
         }
         int lapsCount = deviceResults.size();
+
+        boolean isFastest = false;
+        //check if the current Lap is the fastest lap.
+        isFastest = isFastestLap(deviceResults,lapTime,lapNumber);

```

Comment:
I think we should compare against all results of all pilots in current race.  This way  the first lap might also need to be announced.